src/ape/managers/accounts.py

```diff
@@ -1,12 +1,66 @@
 from typing import Dict, Iterator, List, Type
 
-from ape.api.accounts import AccountAPI, AccountContainerAPI, TestAccountAPI
+from ape.api.accounts import AccountAPI, AccountContainerAPI, ImpersonatedAccount, TestAccountAPI
 from ape.types import AddressType
-from ape.utils import cached_property, singledispatchmethod
+from ape.utils import ManagerAccessMixin, cached_property, singledispatchmethod
 
 from .base import BaseManager
 
 
+class TestAccountContainer(list, ManagerAccessMixin):
```

Comment:
Look into this being the actual `TestAccountContainer` registered OR possibly rename

Reply:
This is actually like a container of containers...

The thing is... What if different ecosystems register their own test accounts (meaning they have their own account containers that yield them)? You would get all those accounts here, so 0-19 are my ethereum test accounts and 20-29 are my starknet ones, etc.

So I am leaning on **not** moving this to `ape-test.TestAccountContainer`

Reply:
Renamed to `TestAccountManager`